test/unit/services/workers/test-generate-payout-payments.js

```diff
@@ -70,7 +70,7 @@ describe('services/workers/generate-payout-payments', function () {
       expect(getClaimsPendingPayment.calledOnce).to.be.true //eslint-disable-line
       expect(getTopUpsPendingPayment.calledOnce).to.be.true //eslint-disable-line
       expect(createPayoutFile.calledWith(claimsPendingPayment)).to.be.true //eslint-disable-line
-      expect(sftpSendPayoutPaymentFile.calledWith(testPath, './test.csv')).to.be.true //eslint-disable-line
+      expect(sftpSendPayoutPaymentFile.calledWith(`/tmp/${testPath}`, './test.csv')).to.be.true //eslint-disable-line
```

Comment:
does this need updating? guess it's not sftp anymore?

Reply:
No, the files are still SFTPed